### DISCUSSION TITLE
Current version of RDBMS is not supported.

### DISCUSSION THREAD

**Discussion by drew-bombard** [markshust/docker-magento#1164] [General]
Anyone else recently getting issues with

`Current version of RDBMS is not supported. Used Version: 10.6.17-MariaDB-1:10.6.17+maria~ubu2004. Supported versions: MySQL-8, MySQL-5.7, MariaDB-(10.2-10.4)`

when running setup:upgrade?  I've been working in my local sandbox for awhile and have a successfully running beta of 2.4.7 on PHP 8.2 and MariaDB 10.6 on an Adobe Commerce Staging server.  But my sandbox has recently been barking this back at me.

I'd prefer to stay on 10.6 but have resorted to bumping it down to 10.4 in my composer.yaml file f

**Comment by YevhenZvieriev** [comment]
Hi, @drew-bombard 

If you'd prefer to stay on 10.6 you should edit file `app/etc/di.xml`.

You can specify acceptable MariaDB versions here - currently, you defined 10.2-10.4

Example to allow MariaDB 10.6:

```
<item name="MariaDB-(10.2-10.6)" xsi:type="string">^10\.[2-6]\.</item>
```
